doc/site/data/configs.json

```diff
@@ -1320,6 +1320,13 @@
     "type": "int"
   },
   "MiniMapDrawProjectiles": {
+    "declarationFile": "/spring/rts/Game/UI/MiniMap.cpp",
```

Comment:
This file is autogenerated by the macro you use to define config vars. It should be removed from git tracking: https://github.com/beyond-all-reason/RecoilEngine/issues/2457